microsoftgraph/microsoft-graph-explorer-v4
# Feature: Enable loading of samples and Permissions descriptions files from Github

## Overview

This PR proposes adding of testing functionality for samples and permissions descriptions files in the DevX Content repo

## Testing Instructions

- Run Graph Explorer locally on your machine
- Clone [https://github.com/microsoftgraph/microsoft-graph-explorer-api](url) and Run DevX API locally
- Override fetching from blob storage by calling and supplying org and branchName params to the devx-api query string
`http://localhost:3000/?devx-api=https://localhost:5001&org=mg&branchName=dev`

- The call should return a set of samples and permissions from the DevX Content repo
